src/print.rs

```diff
@@ -114,13 +115,13 @@ pub fn get_prompt(context: Context) -> String {
 
     let module_strings = root_module.ansi_strings_for_shell(context.shell, Some(context.width));
     if config.add_newline && context.target != Target::Continuation {
-        // continuation prompts normally do not include newlines, but they can
+        // continuation profiles normally do not include newlines, but they can
```

Comment:
I think in this case, it's better to call it the "continuation prompt". That's not naming from our code, it's terminology that's established from the shell. (See e.g. [this answer](https://unix.stackexchange.com/a/193660/379272))

Reply:
Bar replace; will fix